zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageProperties.java

```diff
@@ -253,6 +262,23 @@ public void setPassword(String password) {
     this.password = emptyToNull(password);
   }
 
+  public String getCredentialsFile() {
+    return credentialsFile;
+  }
+
+  public void setCredentialsFile(final String credentialsFile) {
+    this.credentialsFile = credentialsFile;
+  }
+
+  public Integer getCredentialsRefreshInterval() {
```

Comment:
I think we can use `Duration` to support any type of interval like `5s` `1m`

Reply:
lets not. seconds is important. if we allow other units, it will invalidate some things, like what is reasonable (ex millis is not)

Reply:
when this switched to not use poll events, a tradeoff to using old school i/o was to not spin, and seconds gives us that. I know there are a lot of comments so easy to get lost in them

Reply:
Makes sense thanks for the context.